handbook/company/pricing-features-table.yml

```diff
@@ -547,7 +547,7 @@
   productCategories: [Device management]
   pricingTableCategories: [Device management]
 - industryName: Native macOS update reminders
-  documentationUrl: https://fleetdm.com/docs/using-fleet/mdm-macos-updates
+  documentationUrl: https://fleetdm.com/learn-more-about/os-updates
```

Comment:
Use updated doc url instead. Keep "learn-more-about" pure. Only use for within the core product. Makes it easier to understand and avoids nightmares.